patch/llpcPatchDescriptorLoad.cpp

```diff
@@ -79,16 +79,16 @@ PatchDescriptorLoad::PatchDescriptorLoad()
 }
 
 // =====================================================================================================================
-/// Create a non-strided, non-swizzled buffer descriptor.
-///
-/// @param baseAddr (i64) base address of the descriptor (top 16 bits are ignored);
-///                 for convenience, an <N x i32> is also accepted (using the low 48 bits)
-/// @param size (i32) size of the referenced buffer in bytes
+// Create a non-strided, non-swizzled buffer descriptor.
+//
```

Comment:
Please remove all comments for parameters in function header comments. And move them down. You can reference exisiting codes. Write like this:

// === ...
// Create ...
Value* PatchDescriptorLoad::BuildBufferDescriptor(
    Value*      pBaseAddr // [in] Base address of ...
                                       // ...
    Value*       pBufSize    // [in] ..
   Instruction* pInsertPoint) // [in] Where to insert the instructions

Pay attention to the alignment.